README.md

```diff
@@ -1,59 +1,42 @@
 # garden-shoot-trust-configurator
 
-[![reuse compliant](https://reuse.software/badge/reuse-compliant.svg)](https://reuse.software/)
+[![REUSE status](https://api.reuse.software/badge/github.com/gardener/garden-shoot-trust-configurator)](https://api.reuse.software/info/github.com/gardener/garden-shoot-trust-configurator)
 
-## How to use this repository template
 
-TODO(theoddora):
-- Set the repository description in the "About" section of your repository
-- Describe the new component in additional sections in this `README.md`
-- Ask the [Owner of the gardener github organisation](https://github.com/orgs/gardener/people?query=role%3Aowner)
-  - to double-check the initial content of this repository
-  - to create the maintainer team for this new repository
-  - to make this repository public
-  - protect at least the master branch requiring mandatory code review by the maintainers defined in CODEOWNERS
-  - grant admin permission to the maintainers team of the new repository defined in CODEOWNERS
-
-## Maintain copyright and license information
-By default all source code files are under `Apache 2.0` and all markdown files are under `Creative Commons` license.
-
-When creating new source code files the license and copyright information should be provided using corresponding SPDX headers.
-
-Example for go source code files (replace `<year>` with the current year)
-```
-/*
- * SPDX-FileCopyrightText: <year> SAP SE or an SAP affiliate company and Gardener contributors
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-```
+### Get your reuse badge
 
-### Third-party source code
+// TODO(theoddora): Get your reuse badge
+To get your project reuse compliant you should register it [here](https://api.reuse.software/register) using your SAP email address. After confirming your email, an initial reuse check is done by the reuse API.
```

Comment:
Isn't it already done? I see the project is available and compliant at https://api.reuse.software/info/github.com/gardener/garden-shoot-trust-configurator